index.html

```diff
@@ -7,24 +7,27 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Abdul's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 
 <body>
-    <header>
+    <header class="header">
         <div class="wraper-logo">
             <a href="#" class="logo">My Logo</a>
         </div>
         <nav class="menu">
-            <img src="./assets/icons/Union.png" alt="Menu Icon" class="header-image-mobile">
+            <a href="#"><img src="./assets/icons/Union.png" alt="Menu Icon" class="header-image-mobile"></a>
+            <i class="fa fa-close" id="close-icon" style="font-size:36px"></i>
+            
             <ul class="top-nav-desktop">
-                <li><a href="#" class="top-nav-items-desktop">Portfolio</a></li>
-                <li><a href="#" class="top-nav-items-desktop">About</a></li>
-                <li><a href="#" class="top-nav-items-desktop">Contact</a></li>
+                <li><a href="#" class="top-nav-items-desktop" id="porfolio">Portfolio</a></li>
+                <li><a href="#" class="top-nav-items-desktop" id="about">About</a></li>
+                <li><a href="#" class="top-nav-items-desktop" id="contact">Contact</a></li>
```

Comment:
- [ ] Kindy ensure when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. I can't navigate to the about and contact section.